{Release} Upgrade to Azure CLI 2.47.0 (#26004)
Azure/azure-cli: src/azure-cli/HISTORY.rst

Release History
===============

2.47.0
++++++

**AKS**

> * `az aks create/update`: `--tier` argument will specify the sku tier that customer wants (#24940)

Reviewer: ```suggestion
* `az aks create/update`: Add `--tier` argument will specify the sku tier that customer wants (#24940)
```

**Monitor**

* `az monitor activity-log alert`: Adjust help message (#25877)

**NetAppFiles**

* Patch assign snapshotpolicyID (#25789)

**Network**

* [BREAKING CHANGE] `az network`: Clean up irrelevant commands in azure-stack profiles. (#25702)
* [BREAKING CHANGE] `az network application-gateway waf-policy custom-rule`: Rename output property `applicationGatewayIpConfigurations` to `applicationGatewayIPConfigurations` to keep consistent with the name in API (#25837)
* `az network routeserver create/update`: Add parameter `--hub-routing-preference` (#25821)
* Drop `azure-mgmt-network` SDK (#25451)
* Fix #25784: `az network private-link-service update`: `--lb-frontend-ip-configs` cannot be used (#25840)

**RDBMS**

* [BREAKING CHANGE] `az postgres flexible-server replica create`: Fix the behavior of AZ selection in case zone is not passed as parameter (#25843)
* Fix #368903181: Fix zone selection during creation of replica (#25843)
* `az mysql flexible-server restore/geo-restore`: Add parameters to enhance PITR (#25867)
* `az mysql flexible-server replica create`: Add parameters to support cross region paired vnet (#25926)

**Service Bus**

> * `az servicebus namespace authorization-rule keys renew/list` `az servicebus topic authorization-rule keys renew/list` `az servicebus queue authorization-rule keys renew/list`: Parameter `-n` added to create auth_rule_name (#25728)

Reviewer 2: ```suggestion
* `az servicebus namespace/topic/queue authorization-rule keys renew/list`: Add `-n` option for `--authorization-rule-name` to create auth rule (#25728)
```

Release History
===============

2.47.0
++++++

**AKS**

* `az aks create/update`: `--tier` argument will specify the sku tier that customer wants (#24940)

> * `aks nodepool operation-abort`: Add new command to support aborting last running operation on nodepool (#25661)
> * `aks operation-abort`: Add new command to support aborting last running operation on managed cluster (#25661)

Reviewer: ```suggestion
* `az aks nodepool operation-abort`: Add new command to support aborting last running operation on nodepool (#25661)
* `az aks operation-abort`: Add new command to support aborting last running operation on managed cluster (#25661)
```

* `az appconfig kv import`: Ensure the case of imported boolean values does not change for string conversion from file (#25758)

**App Service**

* Fix #25375: `az functionapp deployment source config-zip`: Fix the `Could not find a 'AzureWebJobsStorage' application setting` error (#25698)
* Fix #25876: `az webapp config ssl import`: Fix the UnboundLocalError `local variable 'cert_name' referenced before assignment` (#25891)
* `az functionapp create`: Support container app deployments (#25680)
* `az functionapp delete`: Add a validation to check whether Azure Functions is not in the Azure Container app environments (#25979)

**ARM**

* `az deployment group create`: Support deployment with bicepparam files (#25612)
* `az resource patch`: Add new command to support updating resource by PATCH request (#25746)
* Fix #25706: `az bicep format`: Fix the TypeError `ensure_bicep_installation() missing 1 required positional argument 'cli_ctx'` (#25707)
* Fix #25715: `az bicep install/upgrade`: Fix the `configparser.NoSectionError: No section: 'bicep'` (#25729)

**BREAKING CHANGE**

* Deprecate image alias `Win2008R2SP1`, `UbuntuLTS` (#25930)

**Compute**

> * `vm reimage`: Add new command to support reimaging a virtual machine (#25884)

Reviewer: ```suggestion
* `az vm reimage`: Add new command to support reimaging a virtual machine (#25884)
```

**Cosmos DB**

* `az cosmosdb identity assign`: Allow refreshing user assigned identities if they're reassigned to an account (#25712)

**Extension**

* `az extension add`: Add actionable message for extension not found error (#25921)

> 
> **FabricBot**
> 
> * Update mentions for Azure Deployments team (#25976)

Reviewer 3: ```suggestion
```

**Key Vault**

* `az keyvault region`: GA MHSM region commands (#25942)

**Monitor**

* `az monitor activity-log alert`: Adjust help message (#25877)

**NetAppFiles**

> * Patch assign snapshotpolicyID (#25789)

Reviewer 3: ```suggestion
* `az netappfiles volume update`: Patch assign snapshotpolicyID (#25789)
```

**Network**

* [BREAKING CHANGE] `az network`: Clean up irrelevant commands in azure-stack profiles. (#25702)
* [BREAKING CHANGE] `az network application-gateway waf-policy custom-rule`: Rename output property `applicationGatewayIpConfigurations` to `applicationGatewayIPConfigurations` to keep consistent with the name in API (#25837)
* `az network routeserver create/update`: Add parameter `--hub-routing-preference` (#25821)
* Drop `azure-mgmt-network` SDK (#25451)
* Fix #25784: `az network private-link-service update`: `--lb-frontend-ip-configs` cannot be used (#25840)

**RDBMS**

* [BREAKING CHANGE] `az postgres flexible-server replica create`: Fix the behavior of AZ selection in case zone is not passed as parameter (#25843)

> * Fix #368903181: Fix zone selection during creation of replica (#25843)

Reviewer 2: ```suggestion
* `az postgres flexible-server replica create`: Fix zone selection during creation of replica (#25843)
```
